lib/active_triples/relation.rb

```diff
@@ -40,14 +45,14 @@ def clear
     def result
       parent.query(:subject => rdf_subject, :predicate => predicate)
         .each_with_object([]) do |x, collector|
-          converted_object = convert_object(x.object)
-          collector << converted_object unless converted_object.nil?
+        converted_object = convert_object(x.object)
```

Comment:
I think rubocop barks at the four-line indent, here. I see what you mean, though. Maybe:

```ruby
statements = parent.query(:subject => rdf_subject, :predicate => predicate)
statements.each_with_object([]) do |x, collector|
  ...
end
```